src/components/structures/RoomStatusBar.js

```diff
@@ -289,10 +300,11 @@ module.exports = React.createClass({
         if (hasUDE) {
             title = _t("Message not sent due to unknown devices being present");
             content = _t(
-                "<showDevicesText>Show devices</showDevicesText> or <cancelText>cancel all</cancelText>.",
+                "<showDevicesText>Show devices</showDevicesText>, <sendAnywayText>send without verifying</sendAnywayText> or <cancelText>cancel all</cancelText>.",
```

Comment:
it's not verifying we care about - it's the fact that they are unknown. I worry that this wording adds to the general confusion that you need to have verified devices before e2e will work.

We could try something like "mark devices known and send"

Reply:
yeah, true - let's try that